participans_gallery.js

```diff
@@ -0,0 +1,43 @@
+const COUNT_VISIBLE_PARTICIPANT = 3;
+const WIDTH_ONE_PARTICIPANT = 414;
+let currentScrollPosition = 0;
+
+const listParticipants = document.querySelector('.participants');
+const participants = listParticipants.querySelectorAll('.participant');
+const scrollPageCount = Math.ceil(participants.length);
+const scrollCounter = document.querySelector('.scroll-list-box');
+const currentScrollPage = document.querySelector('.visible-count');
+
+document.querySelector('.total').textContent = scrollPageCount;
+
+let currentPage = COUNT_VISIBLE_PARTICIPANT;
+currentScrollPage.textContent = currentPage;
+
+const buttonRight = scrollCounter.querySelector('.right');
+const buttonLeft = scrollCounter.querySelector('.left');
+
+setButtonDisabled();
+
+
+scrollCounter.onclick = function(event) {
+    const button = event.target.closest('button');
+    if (!button) return;
+
+    if (button.dataset.direction == 'left') {
+        currentPage -= COUNT_VISIBLE_PARTICIPANT;
+        currentScrollPosition += COUNT_VISIBLE_PARTICIPANT * WIDTH_ONE_PARTICIPANT;
+    } else {
+        currentPage += COUNT_VISIBLE_PARTICIPANT;
+        if (currentPage > scrollPageCount) currentPage = scrollPageCount;
+        currentScrollPosition -= COUNT_VISIBLE_PARTICIPANT * WIDTH_ONE_PARTICIPANT;
+    }
+
+    setButtonDisabled();
+    listParticipants.style.marginLeft = currentScrollPosition + 'px';
```

Comment:
```suggestion
    listParticipants.style.marginLeft = `${currentScrollPosition}px`;
```

с интерполяцией получается более выразительно 🤌